src/tools/getPageTool.ts

```diff
@@ -1,53 +1,46 @@
 import { getPage } from '@cosense/std/rest';
-import type { CallToolResult } from '@modelcontextprotocol/sdk/types.js';
+import type { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
 import { isErr, unwrapErr, unwrapOk } from 'option-t/plain_result';
+import { z } from 'zod';
```

Comment:
Good catch! Added `zod` to the dependencies in `package.json` since we're using it directly in the tools.